index.js

```diff
@@ -177,6 +217,60 @@ class Wire extends stream.Duplex {
     this._push(MESSAGE_KEEP_ALIVE)
   }
 
+  sendPe1 () {
```

Comment:
Why are these functions never used?

Would be great to have at least minimum tests

Reply:
https://github.com/webtorrent/webtorrent/pull/1820/files#diff-e5eeda71546bb4a993c32e575d0c54c2ef19adffc968e3f63cf20e623bcedb52R208